### ISSUE TITLE
Complex STFT in X-UMX loss functions (like Danna-Sep)

### ISSUE THREAD

**Issue by sevagh** [sevagh/xumx-sliCQ#8]

**Comment by sevagh** [comment]
Also consider Norbert.

**Comment by sevagh** [comment]
>model 1: A X-UMX model [1] which is initialized with the weights of the official baseline, and is fine-tuned with a modified Combinational Multi-Domain Loss from [1]. In particular, we implement and apply a differentiable Multichannel Wiener Filter (MWF) [2] before the loss calculation, and compute the frequency-domain L2 loss with raw complex values.

https://github.com/yoyololicon/music-demixing-challenge-ismir-2021-entry/tree/5ef1e23104218d484e927b52f5b32db0af9c0e43

**Comment by sevagh** [comment]
I don't need to try this yet, since I want to stick closer to UMX/X-UMX.
